extensions/sentry/src/ProjectDropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import { Icon, List, showToast, Toast } from "@raycast/api";
+import { useEffect } from "react";
+import { useProjects } from "./hooks";
+import { Project } from "./types";
+
+function ProjectDropdownItem(props: { project: Project }) {
+  return (
+    <List.Dropdown.Item
+      value={props.project.slug}
+      title={props.project.slug}
+      icon={{ source: Icon.Circle, tintColor: props.project.color }}
+    />
+  );
+}
+
+export function ProjectDropdown(props: { onProjectChange: (project: Project) => void }) {
+  const { data: projects, error } = useProjects();
+
+  useEffect(() => {
+    if (error) {
+      showToast({ style: Toast.Style.Failure, title: "Failed retrieving projects", message: error.message });
+    }
+  }, [error]);
```

Comment:
@mathieudutour I remember you mentioned to use an effect to show a failure toast. Is this the proper way?

Reply:
Yep that's it 💯